src/bar-chart.js

```diff
@@ -50,6 +51,13 @@ dc.barChart = function (parent, chartGroup) {
         return _chart._render();
     });
 
+    _chart.label(function(d) {
+        return d.y0 + d.y;
+    });
+    //_chart.label() sets _renderLabel to true.
+    //Set back to false to disable renderLabel by default
+    _chart.renderLabel(false);
+
```

Comment:
this seems like an odd default value, the Y coordinate? why not just leave `.label()` uninitialized?